src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -343,8 +384,13 @@ message ParameterBinding {
   // Name for the binding
   string name = 1;
 
+  // Note: Either *data* or *rpc_shared_memory* field will be populated
+
   // Data for the binding
   TypedData data = 2;
```

Comment:
Should we use the oneof semantics?

Reply:
@pragnagopa Suspects that it may not be backward compatible (and hence break other language workers that we don't modify yet) - I will test that theory out later and make a final call on this then.